src/components/Pages/ProfilePage/DeleteAccountForm.js

```diff
@@ -113,65 +119,48 @@ class DeleteAccountFormBase extends React.Component {
         this.props.user.email,
         this.state.password
       );
-      this.props.firebase
-        .auth()
-        .currentUser.reauthenticateWithCredential(cred)
-        .then(() => {
-          this.props.firebase
-            .auth()
-            .currentUser.delete()
-            .then(() => {
-              apiCall("users.delete", { user_id: this.props.user.id }).then(
-                (json) => {
-                  this.props.firebase.auth().signOut();
-                  this.props.reduxLogout();
-                }
-              );
-            });
+      Auth.currentUser.reauthenticateWithCredential(cred).then(() => {
```

Comment:
I just changed this in my BHN-fixes branch.  Need to make sure this works for passwordless.  Do we need to reauthenticate to delete the account?